### ISSUE TITLE
right click part of model and jump to code

### ISSUE THREAD

**Issue by jessfraz** [KittyCAD/modeling-app#3603]
see video

https://github.com/user-attachments/assets/6c982a41-1c31-4c19-8073-4dd7faf62060

**Comment by pierremtb** [comment]
I believe this is being added by @rajgandhi1 in https://github.com/KittyCAD/modeling-app/pull/7664 🥳

**Comment by rajgandhi1** [comment]
yes should I add a shortcut with it like how the other options have it? @jessfraz  
if yes then what keys should I bind?
